### DISCUSSION TITLE
Github Pages build failed (although it passed a few months ago)

### DISCUSSION THREAD

**Discussion by zserge** [community/community#21951] [Other Feature Feedback, Questions, & Ideas]
<p>I was trying to publish an update to my personal Github Page (zserge/zserge.github.io), and received an email about build failure with no furher error descriptions:</p>
<pre><code>The page build failed for the `master` branch with the following error:
Page build failed. For more information, see https://help.github.com/articles/troubleshooting-github-pages-builds/.
For information on troubleshooting Jekyll see:
  https://help.github.com/articles/troubleshooting-jekyll-builds
If you have any questions you can contact us by replying to this email.
</code></pre>
<p>I reverted my repo changes back to the revision where it was last successfully published, and the error is still here. This makes me think that either something has been changed over the last few months, or pages build process got broken somehow.</p>
<p>The repo I’m  referring to is <a href="https://github.com/zserge/zserge.github.io" rel="nofollow noopener">https://github.com/zserge/zserge.github.io</a> and it contains no Jekyll, just plain static HTML/CSS. Any help is highly appreciated.</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p><a class="mention" href="/zserge">@zserge</a> Thanks for being part of the GitHub Community Forum and for asking about this.</p>
<p>As the error message indicates, going through the options in the <a href="https://help.github.com/articles/troubleshooting-github-pages-builds/" rel="nofollow noopener">Help article</a> can help. Because you specifically mentioned that your site is static HTML/CSS with no Jekyll, the easiest fix would be to disable Jekyll by adding a <code>.nojekyll</code> file like described <a href="https://help.github.com/articles/files-that-start-with-an-underscore-are-missing/" rel="nofollow noopener">here</a>.</p>
<p>If that still doesn’t do the trick, then I’d recommend reaching out to us via private support at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> so that we can look a little closer at your account details to see what the problem may be.</p>
<p>Thanks again, <a class="mention" href="/zserge">@zserge</a>! Let me know if you have further questions.</p>
<p>Cheers!</p>
